### PR TITLE
Update Commands

### PR DESCRIPTION
**Pull request by HashWarlock**
## Description
This will be initial release with limited functionality of the following:
- `swanky phala init [PROJECT_NAME]`
- `swanky phala node start`
- `swanky phala node setup`
- `swanky phala contract compile [CONTRACT_NAME]`
- `swanky phala contract deploy [CONTRACT_NAME] `
- `swanky phala contract typegen [CONTRACT_NAME]`
- `swanky phala contract list`
- `swanky phala account list`

### CODE REVIEW

File: src/commands/phala/check.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    force: Flags.boolean({char: 'f'}),
+  }
+
+  static args = [{name: 'file'}]
+
+  public async run(): Promise<void> {
+    const {args, flags} = await this.parse(PhalaCheck)
+
+    const name = flags.name ?? 'world'
+    this.log(`hello ${name} from /home/hashwarlock/Projects/Phala/Swanky/swanky-plugin-phala/src/commands/phala/check.ts`)
```
Comment (Leechael) [inline src/commands/phala/check.ts]: Maybe we need to do some clean up?

**Review by Leechael — COMMENTED** [review]
Also I think it's not a good idea call `yarn` inside each sub-command.

File: src/commands/phala/check.ts (hunk 1 above)
Comment (HashWarlock) [reply src/commands/phala/check.ts]: Removed for better output.

**Comment by HashWarlock** [comment]
Merging then creating new PR for issue #5

**Pull request merged**